deploy/terraform/ldapi/modules/server/outputs.tf

```diff
@@ -0,0 +1,9 @@
+output "self_link" {
+  description = "Self link of the graphql server instance"
```

Comment:
Should these comments be updated to `ldapi`?

Reply:
Yes, thanks!